---
course_id: 7-349-biological-computing-at-the-crossroads-of-engineering-and-science-spring-2005
layout: course_section
menu:
  leftnav:
    identifier: 3373a15daf7a1123e3f742c003f5aaf9
    name: Readings
    weight: 30
title: Readings
type: course
uid: 3373a15daf7a1123e3f742c003f5aaf9

---

| LecÂ # | Topics | Overviews | Readings |
| --- | --- | --- | --- |
|  {{< br >}}{{< br >}} 1 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Introduction {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} We will begin by introducing ourselves and talking about what each of us hopes to get out of the class, our backgrounds and sources of interest in the subject of the seminar. We will then examine the structure of a sample scientific paper, and discuss what one looks for in a good paper. We will conclude with a brief background necessary to understand the papers assigned for next time. {{< br >}}{{< br >}}  |  |
|  {{< br >}}{{< br >}} 2 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Adleman and his Techniques {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} We will look at the original paper that "shook the world" as well as a paper from a distinct area of biology that uses one of the experimental techniques used by Adleman. We will discuss and critique both papers. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Adleman, L. M. "Molecular computation of solutions to combinatorial problems." _Science_ 266, no. 5187 (November 11, 1994): 1021-4. {{< br >}}{{< br >}} Arguello, R., A. L. Pay, A. McDermott, J. Ross, P. Dunn, H. Avaklan, A. M. Little, J. Goldman, and J. A. Madrigal. "Complementary strand analysis: a new approach for allelic separation in complex polyallelic genetic systems." _Nucleic Acids Research_ 25, no. 11 (1997): 2236-2238. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 3 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Self-assembly for Fun and Profit {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Base-pairing of DNA allows for a built-in duplication mechanism. Antiparallel nature of DNA ensures that strands of DNA can only base-pair in one orientation. Are these two features enough to assure that partially single-stranded DNA molecules with complementary regions will assemble according to "directions" programmed into their sequence? We will look at two papers today-one a proof-of-principle of DNA tile self-assembly, and one (five years later) on investigating physical properties of these tiles. We will discuss whether the proof-of-principle is indeed that, and whether knowing physical properties of the molecules makes any difference in designing the tiles for computation. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Winfree, E., F. Liu, L. A. Wenzler, and N. C. Seeman. "Design and self-assembly of two-dimensional DNA crystals." _Nature_ 394, no. 6693 (August 6, 1998): 539-44. {{< br >}}{{< br >}} Sa-Ardyen, P., A. V. Vologodskii, and N. C. Seeman. "The flexibility of DNA double crossover molecules." _Biophysical Journal_ 84 (June 2003): 3829-37. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 4 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} More Self-assembly - Any Logic to it? {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} So now we know that some partially single-stranded DNA molecules can assemble in a predicted configuration. How complicated can that configuration be? Can computing by self-assembly actually take place? Can one build gates? Circuits? Computation histories? Taking a step back, we will also ask how practical such computations may be. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Mao, C., T. H. LaBean, J. H. Relf, and N. C. Seeman. "Logical computation using algorithmic self-assembly of DNA triple-crossover molecules." _Nature_ 407, no. 6803 (September 28, 2000): 493-6. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 5 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Self-assembly - The Way of a Million Wires? {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Continuing our discussion from the previous week, we ask whether the limitations of the self-assembly approach we discussed make the technology more immediately useful in the area of nanofabrication. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Liu, D., S. H. Park, J. H. Reif, and T. H. LaBean. "DNA nanotubes self-assembled from triple-crossover tiles as templates for conductive nanowires." _Proc Natl Acad Sci U.S.A_. 3, no. 101 (January 6, 2004): 717-722. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 6 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Nanodevices {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} One class of nanodevices consists of molecular level machines that operate on and with the help of DNA strands, and are, in fact, constructed of DNA. We will consider two examples of such devices, examine their construction and kinetic properties, as well as potential uses for such devices. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Yurke, B., A. J. Turberfield, A. P. Mills, F. C. Simmel, and J. L. Neumann. "A DNA-fueled molecular machine made of DNA." _Nature_ 406 (August 10, 2000): 605-8. {{< br >}}{{< br >}} Simmel, F. C., B. Yurke, and R. J. Sanyal. "Operation kinetics of a DNA-based molecular switch." _J Nanosci Nanotechnol_ 2, no. 3-4 (July 2002): 383-90. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 7 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Quorum Sensing - Keeping an Eye on Your Neighbor {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Some bacteria, such as Vibrio fischeri, exhibit quorum sensing behavior, whereby "sensing" a certain concentration of self-secreted autoinducer in the culture is the signal to activate transcription and translation of certain genes. This mechanism is maintained through constitutive production of low levels of quorum-sensing signals that allows bacteria to sense the ambient cell density and to induce the expression of specific genes when signal level rises above threshold. We will analyze the classic paper that describes isolating and determining the roles of various genes in one of the operons responsible for bioluminescence. We will also look at how the various parts of the system can be used as components to engineer a bacterial communication system. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Weiss, Ron, and Thomas F. Knight. "Engineered Communications for Microbial Robotics." In _DNA 2000, Lecture Notes in Computer Science_. (DNA Computing: 6th International Workshop on DNA-Based Computers, Leiden, The Netherlands, June 13-17, 2000.) Edited by A. Condon. Vol. 2054. Berlin, Germany: Springer-Verlag GmbH, 2001, pp. 1-16. ISSN: 03029743. {{< br >}}{{< br >}} Engebrecht, J., K. Nealson, and M. Silverman. "Bacterial bioluminescence: isolation and genetic analysis of functions from _Vibrio fischeri_." _Cell_ 32, no. 3 (March 1983): 773-81. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 8 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} The World's Smallest Biological Computational Device {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Guiness book of world records says that the device described in the first paper for today is the world's smallest biological computational device. Do you agree? What is it useful for? What kind of problems can it solve? How fast? We will also try to figure out just how novel the idea of this machine is. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Benenson, Y., R. Adar, T. Paz-Elizur, Z. Livneh, and E. Shapiro. "DNA molecule provides a computing machine with both data and fuel." _Proc Natl Acad Sci U.S.A._ 100, no. 5 (March 4, 2003): 2191-6. {{< br >}}{{< br >}} Szybalski, W. "Universal restriction endonucleases: designing novel cleavage specificities by combining adapter oligodeoxynucleotide and enzyme moieties." _Gene_ 40, no. 2-3 (1985): 169-73. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 9 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Engineered and Naturally-occurring Molecular Switches {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} In a rare example of basic science following the engineering, scientists in the Breaker lab at Yale engineered molecular switches first, and found them in nature second. What are the features that the natural and man-made (evolved) switches share? What is different? How abundant do you think these naturally-occuring switches are? {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Mandal, Maumita, Benjamin Boese, Jeffrey E. Barrick, Wade C. Winkler, and Ronald R. Breaker. "Riboswitches Control Fundamental Biochemical Pathways in Bacillus subtilis and Other Bacteria." _Cell_ 113 (2003): 577-586. {{< br >}}{{< br >}} Tang, J., and R. R. Breaker. "Rational Design of allosteric ribozymes." _Chemistry and Biology_ 4 (June 1997): 453-9. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 10 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Ciliates - Do They Compute? {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Ciliates are protozoa with two nuclei-micronucleus that stores chromosomal DNA, and macronucleus in which the genes are spliced and rearranged into a large number of gene-size chromosomes. The process by which macronucleus is created from a micronucleus is a complicated multi-stepped process. Many researchers argue that this process has considerable computational power. The exact molecular mechanism has not been worked out, but many conjectures have been made. We will consider one such scrambled gene across a number of species and will consider the implications for one of the popular computational models of gene unscrambling mechanism. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Landweber, L. F., T. C. Kuo, and E. A. Curtis. "Evolution and assembly of an extremely scrambled gene." _Proc Natl Acad Sci U.S.A._ 97, no. 7 (March 28, 2000): 3298-303. {{< br >}}{{< br >}} Prescott, D. M., A. Ehrenfeucht, and G. Rozenberg. "Template-guided recombination for IES elimination and unscrambling of genes in stichotrichous ciliates." _J Theor Biol_ 3, no. 222 (June 7, 2003): 323-30. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 11 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Molecular Gates and Circuits {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Computer scientists who begin to learn biology often think of gene regulatory networks in terms of gates and circuits. Two papers assigned for today deal with constructing and evolving molecular circuits. In what ways are these circuits like the electronic ones? In what ways are they different? Is it still fair to call these "circuits?" What are the potential uses of this technology? {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Yokobayashi, Y., R. Weiss, and F. H. Arnold. "Directed evolution of a genetic circuit." _Proc Natl Acad Sci U.S.A._ 99, no. 26 (December 24, 2002): 16587-91. {{< br >}}{{< br >}} Noireaux, V., R. Bar-Ziv, and A. Libchaber. "Principles of cell-free genetic circuit assembly." _Proc Natl Acad Sci U.S.A._ 100, no. 22 (October 28, 2003): 12672-7. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 12 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Student Presentations {{< br >}}{{< br >}}  |  |  |
|  {{< br >}}{{< br >}} 13 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Bridging the Gap - From Building Networks to Deciphering Networks {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} A group of researchers at Boston University has recently gone from engineering genetic network components to analyzing networks occurring in nature to elucidate how they operate. We will look at two of their papers to see how these two approaches complement each other and how advances in one help advance the other. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Gardner, T. S., C. R. Cantor, and J. J. Collins. "Construction of a genetic toggle switch in Escherichia coli." _Nature_ 403, no. 6767 (January 20, 2000): 339-42. {{< br >}}{{< br >}} Gardner, T. S., D. di Bernardo, D. Lorenz, and J. J. Collins. "Inferring genetic networks and identifying compound mode of action via expression profiling." _Science_ 301, no. 5629 (July 4, 2003): 102-5.   {{< br >}}{{< br >}} {{< br >}}